articles/quickstart/spa/angular-beta/_includes/_centralized_login.md

```diff
@@ -0,0 +1,448 @@
+<!-- markdownlint-disable MD041 MD034 MD002 -->
+
+<%= include('../../_includes/_login_preamble', { library: 'Angular 7+' }) %>
+
+This tutorial will guide you in modifying an Angular application to support login using Auth0, that demonstrates how users can log in, log out, and how to view their profile information. It will also show you how to protect routes from unauthenticated users.
+
+::: note
+This tutorial assumes that you're able to use the [Angular CLI](https://cli.angular.io/) to generate new components and services on your project. If not, please place the files where appropriate and ensure that any references to them regarding import paths are correct for your scenario.
+:::
+
+## Create an Angular application
+
+If you do not already have an application in which to implement the SDK, a new one can be generated using the [Angular CLI](https://cli.angular.io/).
+
+Install the CLI:
+
+```bash
+npm install -g @angular/cli
```

Comment:
@joshcanhelp That's... odd. It's a standard npm global package install. Did you get an error message?

Reply:
Command not found

Reply:
@joshcanhelp That's really odd, it's definitely a legit command. It sounds like you have `npm` installed as otherwise you wouldn't be able to run `npm start` either.

I'm not sure how to proceed with this one. Are you able to try uninstalling and reinstalling Node to see if that fixes the issue?